phoenix-tracing-webapp/src/main/webapp/css/nv.d3.css

```diff
@@ -0,0 +1,769 @@
+
```

Comment:
nvd3 library is used for charting. I will try to drop all the libraries which we can manage without.

It's license : https://github.com/novus/nvd3/blob/master/LICENSE.md